src/frontend/js/widgets/UserLogin/index.not.isJoanieEnabled.spec.tsx

```diff
@@ -111,7 +111,7 @@ describe('<UserLogin />', () => {
 
     await userEvent.click(button);
 
-    screen.getByText(user.username);
+    screen.getByText(user.full_name!);
```

Comment:
Here it is equivalent to: `screen.getByText(undefined);`. Is it what you want? 🤔

Reply:
Nope I just say I'm sure that the user has a full_name property defined and I want to retrieve this text in the rendered content.

Reply:
Yep, but on line 56 you wrote `const user: User = UserFactory({ full_name: undefined }).one();` 🤔

Reply:
Oh crap !! I missed the Github code wrapping, nevermind 😂